source/Registry.h

```diff
@@ -87,6 +91,7 @@ class Registry final {
 
   mutable ConcurrentMap<const Method*, Model> models_;
   ConcurrentMap<const Field*, FieldModel> field_models_;
+  ConcurrentMap<std::string, LiteralModel> literal_models_;
```

Comment:
looks like we're iterating over the models in the `get` call anyway so we probably don't gain much from a map... maybe use `ConcurrentSet` instead?